src/Categories/Category/Monoidal/Instance/Endofunctors.agda

```diff
@@ -0,0 +1,131 @@
+{-# OPTIONS --without-K --safe #-}
+module Categories.Category.Monoidal.Instance.Endofunctors {o ℓ e} where
```

Comment:
Personally, I'd have put this in `Categories.Category.Monoidal.Construction` rather than `Instance` since it's parametrized by an ambient category `C`.